lighthouse-core/audits/is-on-https.js

```diff
@@ -31,13 +30,16 @@ class HTTPS extends Audit {
   }
 
   /**
-   * @param {{scheme: string, domain: string}} record
+   * Checks whether the resource was securely loaded.
+   * We special-case data: URLs, as they inherit the security state of their
+   * referring document url. Localhost is also safelisted.
+   *
+   * @param {{scheme: string, domain: string, protocol: string, securityState: function}} record
    * @return {boolean}
    */
   static isSecureRecord(record) {
-    return SECURE_SCHEMES.includes(record.scheme) ||
-           SECURE_SCHEMES.includes(record.protocol) ||
-           SECURE_DOMAINS.includes(record.domain);
+    return record.securityState() === 'secure' || record.protocol === 'data' ||
```

Comment:
does this work on android? IIRC the original reason for not trusting all the security info from devtools and following back to URLs was all the security fails on android devices

Reply:
thanks for calling this out. after double-checking, we are good for android. 

![image](https://user-images.githubusercontent.com/39191/37436399-ffe11260-27a4-11e8-9678-3888cde7f971.png)

turns out its the more global `Security.securityStateChanged`  that's broken on android, but this secState within network is just fine.

Reply:
@patrickhulce now to test a page that embed 'about:blank' to see if securityState is set correctly....

uno momento.